src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -1540,11 +1592,11 @@ void interact(PlayerInteractEvent event) {
 		if (uCarsAPI.getAPI().isuCarsHandlingPlacingCars() && (plugin.API.hasItemCarCheckCriteria() || event.getPlayer().getItemInHand().getType() == Material.MINECART)) {
 			// Its a minecart!
 			Material iar = block.getType();
-			if (ucars.ignoreRails && (iar == Material.RAILS || iar == Material.ACTIVATOR_RAIL 
+			if (ucars.ignoreRails && (iar == Material.RAIL || iar == Material.ACTIVATOR_RAIL 
 					|| iar == Material.POWERED_RAIL || iar == Material.DETECTOR_RAIL)) {
 				return;
 			}
-			if (!PlaceManager.placeableOn(iar.name().toUpperCase(), block.getData())) {
+			if (!plugin.isBlockEqualToConfigIds(roadBlocks, block)) { //replaces placableOn()
```

Comment:
I think it would have been better here to call PlaceManager.placeableOn(Block) rather than breaking the abstraction and checking against the config